stash/test/faucet.test.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, it, beforeEach, expect, vi } from 'vitest'
+import * as faucetService from '../src/service/FaucetService'
+import { ForbiddenException } from '../src/error/Exception'
+
+describe('FaucetService', () => {
+  const toAddress = '0x00000000000000000000000000000000'
+  const captcha = '10000000-aaaa-bbbb-cccc-000000000001'
```

Comment:
This captcha toke test one publicly available on hcaptcha website [here](https://docs.hcaptcha.com/#test-key-set-publisher-or-pro-account) but we can move it to sops if we need. @goncer opinion? stay or move to sops?

Reply:
that's fine, if its public its public :shrug: